src/libcore/num/wrapping.rs

```diff
@@ -12,13 +12,13 @@
 #![unstable(feature = "wrapping", reason = "may be removed or relocated",
             issue = "27755")]
 
-pub use intrinsics::{add_with_overflow, sub_with_overflow, mul_with_overflow};
+use intrinsics::{ add_with_overflow, sub_with_overflow, mul_with_overflow };
```

Comment:
This change needs a `[breaking-change]` tag in the commit log, even if it's unstable stuff. Why remove the pub though?